src/main/resources/db/migration/V1__tables.sql

```diff
@@ -0,0 +1,67 @@
+BEGIN;
+
+DROP TABLE IF EXISTS object_urls CASCADE;
+DROP TABLE IF EXISTS objects CASCADE;
+DROP TABLE IF EXISTS object_log CASCADE;
+DROP TABLE IF EXISTS versions CASCADE;
+
+CREATE TABLE objects
+(
+    id      BIGSERIAL PRIMARY KEY,
+    hash    CHAR(64) NOT NULL,
+    content BYTEA    NOT NULL
+);
+
+CREATE TABLE object_urls
+(
+    url       TEXT PRIMARY KEY,
+    object_id BIGINT NOT NULL,
+    client_id TEXT   NOT NULL,
+    CONSTRAINT fk_objects_urls_to_object
+        FOREIGN KEY (object_id)
+            REFERENCES objects (id) ON DELETE CASCADE
+);
+
+CREATE TABLE object_log
+(
+    id        BIGSERIAL PRIMARY KEY,
+    operation CHAR(3) NOT NULL,
+    url       TEXT    NOT NULL,
+    old_hash  CHAR(64),
```

Comment:
general: Why do we store the hashes as chars instead of `bytea`?

Reply:
Mostly for easier visual representation, not sure it's the best way though.